data/sql/updates/pending_db_world/rev_1625434614739025100.sql

```diff
@@ -0,0 +1,68 @@
+(8509, 0, 0, 0, 20, 0, 100, 0, 3463, 0, 0, 0, 0, 45, 1, 1, 0, 0, 0, 0, 19, 8479, 0, 0, 0, 0, 0, 0, 0, 'Squire Maltrake - On Quest \'Set Them Ablaze!\' Finished - Set Data 1 1');
+
+
+-- Kalaran Windblade
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 8479;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 8479);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(8479, 0, 0, 1, 62, 0, 100, 0, 1321, 0, 0, 0, 0, 26, 3441, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Gossip Option 0 Selected - Quest Credit \'Divine Retribution\''),
+(8479, 0, 1, 0, 61, 0, 100, 0, 1321, 0, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Gossip Option 0 Selected - Close Gossip'),
+(8479, 0, 2, 3, 62, 0, 100, 0, 1323, 2, 0, 0, 0, 11, 19797, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Gossip Option 2 Selected - Cast \'Conjure Torch of Retribution\''),
+(8479, 0, 3, 0, 61, 0, 100, 0, 1323, 2, 0, 0, 0, 72, 0, 0, 0, 0, 0, 0, 7, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Gossip Option 2 Selected - Close Gossip'),
+(8479, 0, 4, 0, 62, 0, 100, 0, 1323, 3, 0, 0, 0, 80, 847900, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Gossip Option 3 Selected - Run Script'),
+(8479, 0, 5, 6, 19, 0, 100, 0, 3454, 0, 0, 0, 0, 50, 149047, 60, 1, 0, 0, 0, 8, 0, 0, 0, 0, -6683.73, -1194.19, 242.02, 0.212059, 'Kalaran Windblade - On Quest \'The Torch of Retribution\' Taken - Summon Gameobject \'Torch of Retribution\''),
+(8479, 0, 6, 0, 61, 0, 100, 0, 3454, 0, 0, 0, 0, 50, 149410, 60, 1, 0, 0, 0, 8, 0, 0, 0, 0, -6683.73, -1194.19, 240.02, 0.447678, 'Kalaran Windblade - On Quest \'The Torch of Retribution\' Taken - Summon Gameobject \'Light of Retribution\''),
+(8479, 0, 7, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 41, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Kalaran Windblade - On Data Set 1 1 - Despawn Instant'),
+(8479, 0, 8, 0, 38, 0, 100, 0, 1, 1, 0, 0, 0, 12, 8480, 3, 68000, 0, 0, 0, 8, 0, 0, 0, 0, -6680.12, -1194.4, 243.255, 3.00197, 'Kalaran Windblade - On Data Set 1 1 - Summon Creature \'Kalaran the Deceiver\'');
+
+-- Respawn time for Kalaran Windblade lowered to 120 it was 900
+UPDATE `creature` SET `spawntimesecs` = 120 WHERE (`ID` = 8479);
+
+
+-- Kalaran the Deceiver
+UPDATE `creature_template` SET `AIName` = 'SmartAI', `InhabitType` = 4, `unit_flags` = 33555264 WHERE `entry` = 8480;
```

Comment:
Would prefer https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Reply:
that's indeed preferred but not mandatory, at least not as long as Keira3 does not generate them that way